src/node/README.md

```diff
@@ -0,0 +1,25 @@
+# artsy/node
+
+This node contains shared [executors](orb-executors) to be used across our node projects.
+
+## Usage example
+
+To reuse the `build` executor
+
+```yaml
+# In your project's .circleci/config.yml
+
+# Using the volatile label is _not_ recommended.
+# Use the version in the comment at the top of node.yml instead.
+
+orbs:
+  node: artsy/node@volatile
+
+jobs:
+  build:
+    executors: node/build
+    steps:
+      # ... add steps here
```

Comment:
Might be worth adding a link to an example, similar to hokusai up above.

Reply:
I'll round back up on this because I have to do the yarn docs anyway.